docker/build_testing_sheet.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:bionic
+
+RUN apt-get update -y
+
+COPY . /kolibri
+
+COPY ./kolibri/.buildkite $GOOGLE_SPREADSHEET_CREDENTIALS
+
+CMD cd /kolibri && \
+    pip install -r requirements/testing_sheet.txt
+
+VOLUME /kolibridist/
```

Comment:
Any particular reason you opted for a VOLUME rather than a runtime bind mount here? @aronasorman is probably in favor of this approach though.

Reply:
A couple reasons - this volume is being declared at build time, meaning it'll be wiped clean with every build and filled with the contents of that directory. It's the earliest implementation of Docker volumes and it's mostly superseded by the current run-time volume creation.

Generally, we're baking a lot of frequently-changed code during build time and at an early layer. Every rebuild would throw out the commands afterwards.

Runtime bind mounts would be more appropriate here (and in other cases I've described to you) because the cache wouldn't get invalidated by an early `COPY`. A later `COPY`  would preserve more cache, but I don't see a benefit in baking `src` files into the image if they don't need to be there (like if we were using the image for a K8s cluster or something). 

We're using docker for the sole purpose of environment isolation here: why fill the image with files that aren't necessary for environment creation?

Reply:
> Runtime bind mounts would be more appropriate here (and in other cases I've described to you) because the cache wouldn't get invalidated by an earlier COPY. A later COPY would preserve more cache, but I don't see a benefit in baking source files into the image if they don't need to be there (like if we were using the image for a K8s cluster or something).

Let's use the bind mount then.